contribution.md

```diff
@@ -0,0 +1,237 @@
+
+```
+  - $type: regex_replace
+    file: 
+    pattern:
+    replacement:
+    count: 
+```
+where
+- `file` name of file where replacement must be executed
+- `pattern` string or regexp ([learn more](https://docs.python.org/2/library/re.html)) to find
+- `replacement` replacement string
+- `count` (*optional*)  maximum number of pattern occurrences to be replaced
+
+
+**`pytorch_to_onnx`** (*optional*)
+
+List of pytorch-to-onnx conversion parameters, see `model_optimizer_args` for details.
+
+**`caffe2_to_onnx`** (*optional*)
+
+List of caffe2-to-onnx conversion parameters, see `model_optimizer_args` for details.
+
+**`model_optimizer_args`**
+
+Conversion parameter, obtained [earlier](#model-conversion), must be specified in this section, e.g.:
+```
+  - --input=data
+  - --mean_values=data[127.5]
+  - --scale_values=data[127.5]
+  - --reverse_input_channels
+  - --output=prob
+  - --input_model=$conv_dir/googlenet-v3.onnx
+```
+> **NOTE:**  no need to specify `framework`, `data_type`, `model_name` and `output_dir` parameters since them are deduced automatically.
+
+**`framework`**
+
+Framework of original model (`caffe`, `dldt`, `mxnet`, `pytorch`, `tf`)
+
+**`license`**
+
+Path to model's license.
+
+----
+*After this step you will obtain **model.yml** file*
+
+### Example
+
+In this [example](models/public/densenet-121-tf/model.yml) classificational model DenseNet-121\*, pretrained in TensorFlow\*, is downloading from GoogleDrive\* as archive.
+
+```
+description: >-
+  This is an Tensorflow\* version of `densenet-121` model, one of the DenseNet
+  group of models designed to perform image classification. The weights were converted
+  from DenseNet-Keras Models. For details see repository <https://github.com/pudae/tensorflow-densenet/>,
+  paper <https://arxiv.org/pdf/1608.06993.pdf>
+task_type: classification
+files:
+  - name: tf-densenet121.tar.gz
+    size: 30597420
+    sha256: b31ec840358f1d20e1c6364d05ce463cb0bc0480042e663ad54547189501852d
+    source:
+      $type: google_drive
+      id: 0B_fUSpodN0t0eW1sVk1aeWREaDA
+postprocessing:
+  - $type: unpack_archive
+    format: gztar
+    file: tf-densenet121.tar.gz
+model_optimizer_args:
+  - --reverse_input_channels
+  - --input_shape=[1,224,224,3]
+  - --input=Placeholder
+  - --mean_values=Placeholder[123.68,116.78,103.94]
+  - --scale_values=Placeholder[58.8235294117647]
+  - --output=densenet121/predictions/Reshape_1
+  - --input_meta_graph=$dl_dir/tf-densenet121.ckpt.meta
+framework: tf
+license: https://raw.githubusercontent.com/pudae/tensorflow-densenet/master/LICENSE
+```
+
+## Documentation
+
+Documentation is very important part of model contribution, it helps to better understand possible usage of the model. Documentation must be named after suggested models name.
+Documentation must contain:
+* description of model
+	* main purpose
+	* features
+	* links to paper or/and source
+* model specification, e.g. type, source framework, GFLOPs and number of parameters
```

Comment:
Is there a way for external contributor to compute model GFLOPs? If not then it should not be treated as mandatory field.